packages/components/src/components/badge/badge.scss

```diff
@@ -0,0 +1,76 @@
+.dc-badge {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    color: #ffffff;
```

Comment:
```suggestion
    color: var( --text-prominent) ?
```

Reply:
hey @thisyahlen-deriv i decided to leave color prop as constant white here as default color because:
1. If we use `Bordered` badge it will apply `color: var(--text-prominent)`  with `border: 1px solid var(--text-prominent)` and the color of text and border will be changed depending on theme (white or black)
2. If we use `Contained`  badge it will apply default `color: white` because in our figma design all contained badges have the same color for dark and light theme, so if i replace the color with your suggestion, for `Contained` badge when i will switch the theme, text will change from white to black.